python/seldon_deploy_sdk/auth/openid.py

```diff
@@ -26,8 +28,21 @@ def __init__(self, config: Configuration):
         )
         self._app.load_server_metadata()
 
-    def authenticate(self, user: str, password: str) -> str:
-        token = self._app.fetch_access_token(
-            username=user, password=password, scope="openid"
-        )
-        return token["access_token"]
+    def authenticate(self, **kwargs) -> str:
+
+        if self._auth_method == 'password_grant':
+            user = kwargs.get('username', None)
+            password = kwargs.get('password', None)
+            password = kwargs.get('scope', 'openid profile email groups')
```

Comment:
is this a typo, should it be:
```suggestion
            scope = kwargs.get('scope', 'openid profile email groups')
```